tools/mo/openvino/__init__.py

```diff
@@ -1,4 +0,0 @@
-# Copyright (C) 2018-2023 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
Why are you removing it?

Reply:
please put details into description of PR. It is also a good practice because you are implementer and can create short reasonable summary. In JIRA we can have a mess with comments, inaccurate description, etc.